lib/localFunction.js

```diff
@@ -6,17 +6,26 @@ var request = require("request");
 var encodeFirestoreValue = require("./firestore/encodeFirestoreValue");
 var utils = require("./utils");
 
-var LocalFunction = function(trigger, urls, controller) {
+var LocalFunction = function(trigger, functionsInfo, urls, controller) {
+  var functionInfo = _.find(functionsInfo, { name: trigger.name });
+
   this.name = trigger.name;
   this.eventTrigger = trigger.eventTrigger;
   this.httpsTrigger = trigger.httpsTrigger;
   this.controller = controller;
+  this.label = _.get(functionInfo, ["labels", "deployment-callable"], "false");
```

Comment:
Revise to:
`const isCallable = _.get(functionInfo, ["labels", "deployment-callable"], "false");`

You're not getting all the labels for that function, you're just checking whether it is a callable